client/tx/tx.go

```diff
@@ -335,43 +370,40 @@ func Sign(txf Factory, name string, tx client.TxBuilder) error {
 	}
 
 	pubKey := key.GetPubKey()
-	sigData := &signing.SingleSignatureData{
-		SignMode:  signMode,
-		Signature: nil,
-	}
-	sig := signing.SignatureV2{
-		PubKey: pubKey,
-		Data:   sigData,
-	}
-	err = tx.SetSignatures(sig)
```

Comment:
would like to have your eyes here @aaronc. I remember we had an offline chat where you said setting txBuilder signatures with `Signature: nil` was  necessary. I deleted those lines, all tests pass.

Reply:
They will fail when we switch to running tests with `SIGN_MODE_DIRECT`. All these tests are still using Amino JSON.